### PR TITLE
Trade expiration time

### PR DESCRIPTION
**Pull request by plondon**

### CODE REVIEW

File: src/trade.js [hunk 1]
```diff
@@ -22,6 +22,8 @@ class Trade {
 
   get createdAt () { return this._createdAt; }
 
+  get expiresAt () { return this._expiresAt; }
```
Comment (Sjors) [inline src/trade.js:26]: Can you add a test? Also, can you set a default value for `_expiresAt` in the constructor? You can make the default `null` if you like; not all partners will have an expiration date for trades.

**Review by Sjors — CHANGES_REQUESTED** [review]
